include/crm/common/actions.h

```diff
@@ -321,14 +321,21 @@ enum pe_action_flags {
 #endif
 };
 
-//! \internal Do not use
+/* enum pe_link_state is currently needed for pe_action_wrapper_t (which is
+ * public) but should be removed when that is refactored to no longer need it at
+ * the next API compatibility break
+ */
+
+//!@{
```

Comment:
Is the `@{`/`@}` syntax for grouping? I figured deprecating the `enum` itself would take care of that

Reply:
Yep, it means the comment will show up in the documentation for each value as well as the enum itself